host/sgx/ocalls.c

```diff
@@ -267,60 +265,6 @@ oe_result_t oe_get_quote_verification_collateral_ocall(
     return result;
 }
 
-#else /* !defined(OE_LINK_SGX_DCAP_QL) */
-
-oe_result_t oe_get_quote_verification_collateral_ocall(
```

Comment:
Can you also update the edl_opt_out test? This ocall was tested by the test and expects the `OE_UNSUPPORTED`.

Reply:
The opt our behavior is not affected by this PR. The weak symbol `_oe_get_quote_verification_collateral_ocall` is still retained and that will return `OE_UNSUPPORTED`.

Reply:
ah, got it. It's a different one.